### PR TITLE
fixes fonts on android

### PR DESCRIPTION
**Pull request by chrisnojima**
Alrite,

This was a PITA. Basically the fonts are inconsistent on the platforms due to a couple of things.

1. There's no support in RN for letterSpacing. So the widths won't ever be 100% the same. Spoke to cecile about workarounds and she feels like its subtle enough to not matter
2. The fonts themselves have this thing called vertical metrics which cause where things to be with things like defined line-heights to vary platform to platform. I ran the fonts through https://www.fontsquirrel.com/tools/webfont-generator to remove this.

This did cause some slight changes in the font offsets and things in the app. I looked at the style guide and dumb component sheet and parts of the login / provision flow to see that its looking ok. 

Please take a look around and let me know if something is totally broken. I think after this things will be more sane

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
Also i renamed the fonts to align with what android uses since it has special rules. now there's just one set (but in 2 places cause android needs it in a very specific place)

File: shared/.flowconfig [hunk 1]
```diff
@@ -10,6 +10,8 @@
 .*/node_modules/react-tap-event-plugin/node_modules/.*
 .*/react/node_modules/.*
 ../desktop/build/.*
+../desktop/shared/.*
```
Comment (chrisnojima) [inline shared/.flowconfig]: ignore these symlinks, we pull in ./ anyways

File: shared/common-adapters/text.native.js [hunk 2]
```diff
@@ -186,6 +186,7 @@ const sizeGroups: { [key: '40' | '30' | '22' | '18' | '15' | '14']: Object } = {
 
 const textCommon = {
   ...globalStyles.fontRegular,
+  textAlignVertical: 'center',
```
Comment (chrisnojima) [inline shared/common-adapters/text.native.js]: this only applies to android and matches what ios does by default. with this off the text labels in the stylesheet headers will be top aligned vs middle for example

File: shared/more/style-sheet.render.native.js [hunk 3]
```diff
@@ -130,9 +130,16 @@ const Fonts = () => (
           <Text backgroundMode={backgroundMode} type='HeaderJumbo'>Header Jumbo</Text>
           <Text backgroundMode={backgroundMode} type='HeaderJumbo'>Header Jumbo</Text>
           <Space/>
+          <Box style={{width: 100, height: 100, backgroundColor: 'blue'}} />
```
Comment (chrisnojima) [inline shared/more/style-sheet.render.native.js]: this is just to sanity check sizes on both platforms

File: shared/more/style-sheet.render.native.js [hunk 4]
```diff
@@ -130,9 +130,16 @@ const Fonts = () => (
...
           <Space/>
           <Text backgroundMode={backgroundMode} type='HeaderBig'>Header big Header big</Text>
-          <Text backgroundMode={backgroundMode} type='HeaderBig'>Header big Header big</Text>
+          <Text style={{letterSpacing: undefined, backgroundColor: 'red', textAlignVertical: 'auto'}} backgroundMode={backgroundMode} type='HeaderBig'>Header big Header big</Text>
```
Comment (chrisnojima) [inline shared/more/style-sheet.render.native.js]: just so we can see what no letterspacing looks like on both sides

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**